listenbrainz/webserver/static/js/src/user-feed/UserFeed.tsx

```diff
@@ -405,8 +407,122 @@ export default class UserFeedPage extends React.Component<
     }
   };
 
+  hideFeedEvent = async (event: TimelineEvent) => {
+    const { currentUser, APIService } = this.context;
+    const { newAlert } = this.props;
+    const { events } = this.state;
+
+    try {
+      const status = await APIService.hideFeedEvent(
+        event.event_type,
+        currentUser.name,
+        currentUser.auth_token as string,
+        event.id!
+      );
+
+      if (status === 200) {
+        newAlert("success", "", <>Successfully hidden</>);
+        const new_events = events.map((traversedEvent) => {
+          if (
+            traversedEvent.event_type === event.event_type &&
+            traversedEvent.id === event.id
+          ) {
+            // eslint-disable-next-line no-param-reassign
+            traversedEvent.hidden = true;
+          }
+          return traversedEvent;
+        });
+        this.setState({ events: new_events });
+      }
+    } catch (error) {
+      newAlert("danger", "", <>Could not hide event</>);
```

Comment:
This could potentially be useful:
```suggestion
      newAlert("danger", error.toString(), <>Could not hide event</>);
```